documentation/scripts/documentation_config.ts

```diff
@@ -718,9 +725,16 @@ export const Examples: Example[] = [
         content: 'A formula that return an external image. This sample returns a random photo of a cat.',
         codeFile: './examples/cats/formula.ts',
       },
+      {
+        name: 'Image result from temporary URL',
+        content:
+          'A formula that returns an image uploaded to `temporaryBlobStorage`. This sample returns a random avatar using an API that returns SVG code used to generate an avatar. You could also imagine locally creating an SVG or other image and uploading it to `temporaryBlobStorage`.',
```

Comment:
What do you mean "locally" in this context? On the local machine or in the Pack code itself? May be worth rephrasing to make that clearer.